evidence/pool.go

```diff
@@ -197,6 +203,12 @@ func (evpool *Pool) CheckEvidence(evList types.EvidenceList) error {
 		ok := evpool.fastCheck(ev)
 
 		if !ok {
+			// check that the evidence isn't already committed
+			if evpool.isCommitted(ev) {
+				evpool.logger.Debug("Evidence was already committed, ignoring this one", "ev", ev)
+				return nil
+			}
+
```

Comment:
Don't we need to check the other evidence in the list?

Reply:
actually this should return an error. Wasn't thinking